src/Voxel.MiddyNet/MiddyNet.cs

```diff
@@ -70,6 +93,9 @@ private void InitialiseMiddyContext(ILambdaContext context)
             }
 
             MiddyContext.AdditionalContext.Clear(); //  Given that the instance is reused, we need to clean the dictionary.
+            MiddyContext.MiddlewareBeforeExceptions = new List<Exception>();
```

Comment:
Couldn't we use Clear() ?

Reply:
No, at this point MiddleWareBeforeExceptions is null.